### DISCUSSION TITLE
how to show validation errors from an inactive tabs?

### DISCUSSION THREAD

**Discussion by ghost** [twbs/bootstrap#35849] [Q&A]
Hi,

I have a form with two tabs and a save button (outside the tab). Both tabs have a textbox with required validation. Somehow, the validation error does not fire when user has not fill up all the required inputs and click on save button. Meaning, if I am on the first tab and ONLY fill up the first tab input field and press save. There is no error message coming from the second tab.

So, Is there a way to show validation errors from an inactive tab in v5?

Cheers

**Comment by 035365** [comment]
[file.zip](https://github.com/twbs/bootstrap/files/8092247/file.zip)
